src/components/Settings/EventLog/types.ts

```diff
@@ -0,0 +1,15 @@
+export type Event = {
+  after?: Record<string, any>;
+  before?: Record<string, any>;
+  data?: Record<string, any>;
```

Comment:
I'm wondering if it's possible to tighten up the types here! i tried 
```suggestion
  after?: Record<string, EventValue>;
  before?: Record<string, EventValue>;
  data?: Record<string, EventValue>;
```
<br/>
and then I updated EventValue to:
<br/>


```
export type EventValue = boolean | string | Array<string | boolean | object> | object;
```

im not sure that it's correct but it seemed to work 🥹